rero_ils/modules/contributions/tasks.py

```diff
@@ -83,3 +84,33 @@ def create_mef_record_online(ref):
     if contribution:
         pid = contribution.get('pid')
     return pid, online
+
+
+@shared_task(ignore_result=True)
+def update_contributions(pids=[], dbcommit=True, reindex=True, verbose=False,
```

Comment:
It isn't good to use a default empty array as default value because it's compiled once into the `.pyc`
A better approach is :
```python
def update_contributions(pids=None, dbcommit=True, reindex=True, verbose=False, timestamp=True):
    """docstring."""
    pids = pids or Contribution.get_all_pids()
    ...
```

Reply:
You are absolutely right.